data/sles4sap/agama/sles_sap_default_staging_pxe.jsonnet

```diff
@@ -0,0 +1,74 @@
+local repo = '{{INCIDENT_REPO}}';
+local urls = std.split(repo, ",");
+{
+  product: {
+    id: '{{AGAMA_PRODUCT_ID}}',
+    registrationCode: '{{SCC_REGCODE_SLES4SAP}}',
+  },
+  root: {
+    password: '$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/',
+    hashedPassword: true,
+    sshPublicKey: 'enable ssh',
+  },
+  software: {
+    packages: [],
+    extraRepositories:
+      if repo != "" then
+        [
+          {
+            alias: "TEST_" + std.toString(i),
+            url: urls[i],
+            allowUnsigned: true
+          }
+          for i in std.range(0, std.length(urls) -1)
+        ]
+      else
+        [],
+    onlyRequired: false
+  },
+  questions: {
+    policy: 'auto',
+    answers: [
+      {
+        answer: 'Trust',
+        class: 'software.import_gpg'
+      }
+    ]
+  },
+  storage: {
+    drives: [
+      {
+        search: '/dev/sda',
+        partitions: [
+          { search: '*', delete: true },
+          { generate: 'default' },
+        ],
+      },
+    ],
+  },
```

Comment:
Do we need to set `/dev/sda` as the install device here?

Reply:
I see, then it is known issue on the system with Legacy mode, it hardcode the boot device to first hdd, then can you please rename the agama profile name like `sles_sap_default_staging_pxe_ipmi.jsonnet` ? :)